sdk/keyvault/azure-security-keyvault-keys/src/test/java/com/azure/security/keyvault/keys/KeyClientTestBase.java

```diff
@@ -337,7 +337,8 @@ static void assertKeyEquals(CreateKeyOptions expected, KeyVaultKey actual) {
     public String getEndpoint() {
         final String endpoint = interceptorManager.isPlaybackMode()
                 ? "http://localhost:8080"
-                : System.getenv("AZURE_KEYVAULT_ENDPOINT");
+                : "https://cameravault.vault.azure.net";
+//                : System.getenv("AZURE_KEYVAULT_ENDPOINT");
```

Comment:
remove vault end point url here and below

Reply:
updated